ddprof-lib/src/main/cpp/profiler.cpp

```diff
@@ -1143,7 +1143,7 @@ Error Profiler::start(Arguments& args, bool reset) {
 
         _state = RUNNING;
         _start_time = time(NULL);
-        _epoch++;
+        __atomic_add_fetch(&_epoch, 1, __ATOMIC_SEQ_CST);
```

Comment:
I am not sure how important it is to put a SEQ_CST here. What is the use of epoch ?

Reply:
it's only used to check the recording hasn't rolled since a wallclock sample was taken

Reply:
Ok, that means if we do not retrieve the correct value, we are fine with it. I will go back to no ordering constraints:
 `__ATOMIC_RELAXED`